portality/templates/openurl/400.html

```diff
@@ -0,0 +1,15 @@
+{% extends "/base.html" %}
```

Comment:
Hmm. 400? That's Bad Request (so wrong format of supplied info) rather than Not Found (using supplied info) I'd think.